__init__.py

```diff
@@ -135,13 +150,32 @@ def handle_bender_intent(self, _):
 
     @intent_handler(IntentBuilder("glados_intent").require("glados_keyword").build())
     def handle_glados_intent(self, _):
-        path, files = self.get_reference_files("/sounds/glados", "mp3")
+        path, files = self.get_reference_files("sounds/glados", "mp3")
         if len(files):
             mp3 = path + "/" + random.choice(files)
-            self.play_audio(mp3)
+            if self.ocp:
+                self._play_in_ocp(mp3, title="GlaDOS says...")
+            else:
+                self.log.warning("OCP not available, playing locally, this is not interruptible so consider enabling OCP")
+                self.play_audio(mp3)
         else:
             self.speak_dialog("bad_file")
 
     @skill_api_method
     def get_display_date(self):
         return StarDate().getStardate()
+
+    def _play_in_ocp(self, media, title="Easter Egg!"):
+        data = {
```

Comment:
consideration, if running in containers the path to the mp3 file wont be available inside ovos-audio

you can check for 
` os.environ.get("IS_OVOS_CONTAINER") ` and in this case perhaps send the url for the file from github? that would cause OCP to stream it instead of hiting a bad file path

Reply:
Opened an issue to follow up with that, good callout